includes/php.inc

```diff
@@ -1,6 +1,10 @@
+	# prevent PHP path-walking exploit, remove if PHP files are on remote servers
```

Comment:
Not sure I understand this, or the part if PHP files are on remote servers.  Any harm having this in there if the PHP files are on a separate server we pass to with fastcgi?